app-backend/http4s-util/src/main/scala/com/rasterfoundry/http4s/xray/UdpClient.scala

```diff
@@ -0,0 +1,28 @@
+package com.rasterfoundry.http4s.xray
+
+import java.net._
+
+import cats.effect.Sync
+import io.circe.syntax._
+import io.circe.Printer
+
+class UdpClient[F[_]: Sync] {
+
+  val printer = Printer(
+    preserveOrder = true,
+    dropNullValues = true,
+    indent = ""
+  )
+
+  val address = new InetSocketAddress("xray.service.internal", 2000)
+  val socket = new DatagramSocket()
+
+  def write(segment: Segment[F]): F[Unit] = Sync[F].delay {
```

Comment:
I forgot about these summoners from cats/cats-effect and probably should have used them in #5167 :man_facepalming: